### DISCUSSION TITLE
Using base model avatar + textures VS different avatars

### DISCUSSION THREAD

**Discussion by camelman90** [mozilla/hubs#3107] [Avatar Questions] [answered]
Is using a base model + different textures more performant than using multiple models without textures?

Is there some sort of optimization on the backend to use less drawcalls if the base model is the same?

**Comment by cvanmeurs** [comment]
My understanding is that you want to use as few materials as possible to reduce draw calls.  Try to unwrap everything to one UV space using a single diffuse texture and material for best performance.

**Comment by camelman90** [comment, marked as answer]
Dom (@netpro2k) in Discord:
@hoppy (camelgod) nope. All that happens with the textures there is they will get swapped on the first material of the first mesh in the file to allow for re-skinning. If anything they are maybe less performant, since they are fetched as a separate http request instead of part of the .bin file.. but thats pretty negligible
